src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```diff
@@ -293,37 +296,60 @@ public void setBasePath(String basePath) {
     }
 
     @Override
-    public ProjectResultReport postProject(File file, boolean async, HttpBasicAuth auth, String testCaseName, String testSuiteName, String environment) throws ApiException {
```

Comment:
we could keep and deprecate the old methods and simply delegate internally to the new ones?